lib/marshal-evictions-by-week/viz.tsx

```diff
@@ -0,0 +1,235 @@
+
+type MarshalEvicsVizProps = {
+  fieldName: keyof MarshalEvicsNumericFields,
+  title: string,
+  height: number,
+};
+
+const MarshalEvicsViz: React.FC<MarshalEvicsVizProps> = (props) => {
+  const [timeUnit, setTimeUnit] = useState<MarshalEvicsTimeUnit>("yearweek");
+
+  return (
+    <div>
+      <form>
+        <input type="radio" name="citywide-timeunit" id="citywide-yearweek" 
+        checked={timeUnit==="yearweek"}
+        onChange={e => setTimeUnit("yearweek")}/>
+        <label htmlFor="citywide-yearweek">Week</label>
+
+        <input type="radio" name="citywide-timeunit" id="citywide-yearmonth" 
+        checked={timeUnit==="yearmonth"}
+        onChange={e => setTimeUnit("yearmonth")}/>
+        <label htmlFor="citywide-yearmonth">Month</label>
+      </form>
+      <JsonLoader<MarshalEvicsRow[]> url={MARSHALEVICS.json} fallback={<VizFallback className={VIZ_TIME_SERIES_CLASS} />}>
+        {(values) => <MarshalEvicsVizWithValues values={values} timeUnit={timeUnit} {...props} />}
+      </JsonLoader>
+    </div>
+  );
+};
+
+function thousands_separators(num: any)
+  {
+    var num_parts = num.toString().split(".");
+    num_parts[0] = num_parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+    return num_parts.join(".");
+  }
+
+const MarshalEvicsVizWithValues: React.FC<MarshalEvicsVizProps & {
+  values: MarshalEvicsRow[],
+  timeUnit: MarshalEvicsTimeUnit
+}> = ({values, fieldName, title, timeUnit, height}) => {
+  values = values.filter(
+    // If we are viewing data by week, let's grab data since the first Sunday of Jan 2020
+    // Otherwise, we can grab data from 1/1/2020 onwards
+    row => row.week_eviction >= (timeUnit === "yearweek" ? "2022-01-01 00:00:00" : "2021-12-31 00:00:00")
+    );
+      const evictionsSinceJan = values.filter(
+    row => row.week_eviction >= "2022-01-14 00:00:00"
+  ).reduce(
+    (total, row) => total + row[fieldName], 0
+  );
+
+  // const MarshalEvicsDataLagStart = getMarshalEvicsLagDate(values, 30); // 4 weeks for lag
+  // const MarshalEvicsDataLagEnd = getMarshalEvicsLagDate(values, 0); // latest date
+  const timeUnitLabel = timeUnit === "yearmonthdate" ? "Day"
+    : timeUnit === "yearweek" ? "Week" 
+    : "Month";
+  const barColor = "#B73A3A"
+  const selectedBarColor = "#AF2525";
+  // const lineTop = 20;
+  // const lineBottom = 20;
+  
+
+  // const casesCovidStart = values.find(datapoint => datapoint.day === '2020-03-16T04:00:00.000Z')?.judgments;
+  // const casesCovidStartThousands = thousands_separators(casesCovidStart);
+
+  const spec: VisualizationSpec = {
+    $schema: "https://vega.github.io/schema/vega-lite/v4.json",
+    description: title,
+    width: "container",
+    height: 400,
+    title: {
+      align: "left",
+      anchor: "start",
+      text: `${title}`,
+      fontSize: 24,
+      subtitle: [
+        `There have been ${evictionsSinceJan} evictions by court-ordered marshals since`,  `eviction protections expired on January 15, 2022.`,
```

Comment:
Maybe use the formatting here as well? 

```suggestion
        `There have been ${thousands_separators(evictionsSinceJan)} evictions by court-ordered marshals since`,  `eviction protections expired on January 15, 2022.`,
```